ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_lite.cc

```diff
@@ -729,7 +730,8 @@ TFLiteInterpreter::setInputTensorsInfo (const GstTensorsInfo *info)
      * iterate over all possible ranks starting from MIN rank to the actual rank
      * of the dimension array. In case of none of these ranks work, return error
      */
-    for (int rank = 0 ; rank < NNS_TENSOR_RANK_LIMIT ; rank++) {
+    input_rank = gst_tensor_info_get_rank (tensor_info);
+    for (int rank = input_rank; rank < NNS_TENSOR_RANK_LIMIT; rank++) {
```

Comment:
I want to know if it is "input_rank -1" or not.

Reply:
Updated to use just `rank` instead of `rank+1`
So, It will be okay to use `input_rank`.
Thank you :)

Reply:
Thank you.